frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalettePreview/CommandPreview.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from 'react'
+import styles from './CommandPalettePreview.module.css'
+
+type Props = {
+  commandName: string
+}
+
+const COMMAND_VIDEO_SOURCE: Record<string, string> = {
+  'copy link':
+    'https://assets.liambx.com/erd-core/2025-09-01/videos/copy-link.mp4',
+  'Zoom to Fit':
+    'https://assets.liambx.com/erd-core/2025-09-01/videos/zoom-to-fit.mp4',
+  'Tidy Up': 'https://assets.liambx.com/erd-core/2025-09-01/videos/tidy-up.mp4',
+}
+
+const COMMAND_IMAGE_SOURCE: Record<string, string> = {
+  'Show All Fields':
+    'https://assets.liambx.com/erd-core/2025-09-01/images/show-all-fields.png',
+  'Show Key Only':
+    'https://assets.liambx.com/erd-core/2025-09-01/images/show-key-only.png',
+  'Show Table Name':
+    'https://assets.liambx.com/erd-core/2025-09-01/images/show-table-name.png',
+}
```

Comment:
I made maps to refer to the assets which is set in https://github.com/liam-hq/liam/pull/3326
I think the assets are the same for any environments, so that I didn't consider to use environment variable, as like `${process.env.CDN_BASE_URL}/erd-core/...`.

Reply:
Yes, I think this is enough for now since we are in the same environment.